README.md

```diff
@@ -124,8 +124,12 @@ For a full list of available options, use `-h`:
 ```sh
 ./aws-es-proxy -h
 Usage of ./aws-es-proxy:
+  -assume string
+        Optionally specify role to assume
```

Comment:
Annotation: I re-ran the proxy with `-h` and I guess this option had not been documented yet either.